src/audio/output.rs

```diff
@@ -0,0 +1,35 @@
+//! Provides structures and functions used to get audio outputs.
+
+// We have to use types from this to provide an output iterator type.
+extern crate cpal;
```

Comment:
Somehow my comment was marked as outdated... Could you please move this declaration to `src/lib.rs`?